src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferGatewayIntegrationTest.java

```diff
@@ -17,21 +17,20 @@
 import org.apache.commons.lang.StringUtils;
 import org.awaitility.Duration;
 import org.jetbrains.annotations.NotNull;
```

Comment:
Just out of curiousity, what is a jetbrains annotation?

Reply:
Not sure.